### ISSUE TITLE
Forking stuck when destination path already exists

### ISSUE THREAD

**Issue by StephenKing** [gitblit/gitblit#944]
Given a user has already a repository with the name that a to-be-created fork will also have, the UI will be stuck with "Forking..".
The error message that the repository already exists is only visible in the log file:

````
[ERROR] Failed to fork foo/bar.git for john.doe
com.gitblit.GitBlitException: org.eclipse.jgit.api.errors.JGitInternalException: Destination path "bar.git" already exists and is not an empty
directory
        at com.gitblit.manager.GitblitManager.fork(GitblitManager.java:221)
        at com.gitblit.wicket.pages.ForkPage$ForkThread.run(ForkPage.java:101)
Caused by: org.eclipse.jgit.api.errors.JGitInternalException: Destination path "bar.git" already exists and is not an empty directory
        at org.eclipse.jgit.api.CloneCommand.init(CloneCommand.java:152)
        at org.eclipse.jgit.api.CloneCommand.call(CloneCommand.java:132)
        at com.gitblit.manager.GitblitManager.fork(GitblitManager.java:212)
        ... 1 more
```
Instead, a meaningful error message (or a page to change the destination name) would be helpful.

**Comment by gitblit** [comment]
I added logic to prevent the display of the fork button if the target repo already exists.

**Comment by yakcamnevets** [comment]
Hello, I see that this issue is closed but I am receiving the same error on 1.8.0.

```
2018-07-11 14:58:50 [ERROR] Failed to fork xxx for xxx
com.gitblit.GitBlitException: org.eclipse.jgit.api.errors.JGitInternalException: Destination path "xxx" already
exists and is not an empty directory
        at com.gitblit.manager.GitblitManager.fork(GitblitManager.java:228)
        at com.gitblit.wicket.pages.ForkPage$ForkThread.run(ForkPage.java:101)
Caused by: org.eclipse.jgit.api.errors.JGitInternalException: Destination path "horizon-liberty.git" already exists and is not a
n empty directory
        at org.eclipse.jgit.api.CloneCommand.init(CloneCommand.java:159)
        at org.eclipse.jgit.api.CloneCommand.call(CloneCommand.java:133)
        at com.gitblit.manager.GitblitManager.fork(GitblitManager.java:219)
        ... 1 more
```
